bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -227,7 +227,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <repartition_rain_snow               >.true.</repartition_rain_snow>
 <repartition_rain_snow phys="clm4_5" >.false.</repartition_rain_snow>
 
-<glcmec_downscale_longwave>.true.</glcmec_downscale_longwave>
+<glcmec_downscale_longwave>.false.</glcmec_downscale_longwave>
```

Comment:
@ekluzek 
aux_clm seems promising so far, so I requested your review.

Reply:
The question is if we change it for all physics version, or just clm6_0. I think we should just do this for clm6_0, which would allow you to show that answers don't change for the other physics modes. This also keeps our older physics as close as possible to the settings they had when they were set.

It's also good that you are currently testing it for all physics, because that will ensure it doesn't cause problems for any of our tests in case we decide to do it for all tests.

Reply:
I'd be inclined to just do this for CLM6 physics to avoid changing answers in CLM5.

Reply:
I agree with @wwieder especially after seeing it was set this way for both clm4_5 and clm5_0.

Reply:
I also checked and didn't think there is documentation that needs to be updated.

But, I think you should add a comment above this that talks about what Adam and Gunter found. This is more or less an advanced option, typically we turn on newer options with newer physics, but in this case we are doing the opposite of the normal flow. Here we are turning a more complex option off for the newer physics. So it needs a comment to explain why we are doing that.